manifests/g/GorrilaDevs/Ferium/4.5.0/GorrilaDevs.Ferium.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: GorrilaDevs.Ferium
+PackageVersion: 4.5.0
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: ferium.exe
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+UpgradeBehavior: install
+ReleaseDate: 2024-01-30
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/gorilla-devs/ferium/releases/download/v4.5.0/ferium-windows-msvc.zip

```

Comment:
Is it possible to use the `PackageVersion` variable here?

Reply:
If you think about future updates, the action I have linked (https://github.com/vedantmgoyal2009/winget-releaser) is able to handle that as well.